PocketKit/Sources/PocketKit/MyList/ItemsList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -106,9 +106,11 @@ class SavedItemsListViewModel: NSObject, ItemsListViewModel {
         listOptions
             .objectWillChange
             .dropFirst()
-            .receive(on: DispatchQueue.main).sink { [weak self] _ in
+            .receive(on: DispatchQueue.global(qos: .userInitiated)).sink { [weak self] _ in
```

Comment:
Always ensure that we handle any processing when the list options change on a background, user Initiated thread. See https://developer.apple.com/documentation/dispatch/dispatchqos/qosclass I chose based on what the code does. I.E. the User did initiate it, but we dont need to block on it.